tests/gocase/unit/introspection/introspection_test.go

```diff
@@ -40,6 +40,24 @@ func TestIntrospection(t *testing.T) {
 	rdb := srv.NewClient()
 	defer func() { require.NoError(t, rdb.Close()) }()
 
+	t.Run("TIME", func(t *testing.T) {
+		nowUnix := int(time.Now().Unix())
+
+		r := rdb.Do(ctx, "TIME")
+		vs, err := r.Slice()
+		require.NoError(t, err)
+
+		s, err := strconv.Atoi(vs[0].(string))
+		require.NoError(t, err)
+		require.Greater(t, s, nowUnix - 2)
+		require.Less(t, s, nowUnix + 2)
```

Comment:
No worry about the Go fmt, we can auto-fix it in the future.

```suggestion
		require.Greater(t, s, nowUnix-2)
		require.Less(t, s, nowUnix+2)
```